JialinShi/bulletin-board-frontend
# Add UI for NotePage, UserPage and Navigator bar

# Description
- Added Navigator bar for switching between Note page and Users page
- UI enhancement for Note page and User page
- Added author selector for selecting users as current user for note operations

# Test 
- Tested webpage functionalities including 
  - Create, Update and delete Note
  - Create, Update and delete User
- UI enhancement works as expected 
![image](https://github.com/user-attachments/assets/5e88bedc-9eaf-4993-8203-135fb42f8d15)
![image](https://github.com/user-attachments/assets/7bbac4ab-ba6e-4a31-a7b5-92144407e937)
